src/dmd/backend/cgelem.d

```diff
@@ -5285,6 +5285,13 @@ private elem * elvalist(elem *e, goal_t goal)
 {
     assert(e.Eoper == OPva_start);
 
+    if (funcsym_p.ty() & mTYnaked)
+    {   // do not generate prolog
+        el_free(e);
+        e = el_long(TYint, 0);
```

Comment:
this is a 0 literal, no?

Reply:
I'm 99% sure that this is a kind of noop, as when using the elxxx table there's always an elem expected as result, but later it's eliminated. I want to learn a bit the backend, a confirmation would be nice.

Also look at elinfo(), same trick already used.